PageViews/PageViews.js

```diff
@@ -240,7 +240,7 @@ const PageViewsBase = kind({
 					component={ViewManager}
 					duration={400}
 					index={index}
-					noAnimation={noAnimation}
+					noAnimation={(typeof ENACT_PACK_NO_ANIMATION !== 'undefined' && ENACT_PACK_NO_ANIMATION) || noAnimation}
```

Comment:
Let the `ENACT_PACK_NO_ANIMATION` have higher priority.

```suggestion
					noAnimation={noAnimation || (typeof ENACT_PACK_NO_ANIMATION !== 'undefined' && ENACT_PACK_NO_ANIMATION)}
```